src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -225,14 +225,18 @@ protected function createConfigurationDefinition()
                 ->setAllowedValues(['all', 'namespaced'])
                 ->setDefault('all')
                 ->getOption(),
+            (new FixerOptionBuilder('strict', 'Whether leading ``\`` of constant invocation not meant to have it should be removed.'))
```

Comment:
`` ` `` should not be doubled here (they get tripled in the README file).

Reply:
Of course 🤦‍♂ - fixed.